frontend/components/ModelForm.tsx

```diff
@@ -145,92 +146,30 @@ export const ModelTable = ({
     usePagination,
   )
 
+  const filterOptions = [
+    {
+      label: 'email',
+      options: ['bfranklin@seas.upenn.edu', "ajackson@sas.upenn.edu"],
+      
+    },
+    {
+      label: 'name',
+      options: ['Thomas Jefferson', "James Madison"],
+      
+    },
+
+    
+  ]
```

Comment:
This seems to be hardcoded, are there still changes we need to make before merging?

Reply:
Yeah, I will change this. I used this to test how filtering works.

Reply:
Yes, I will fix these before merging, just to test.